baseplate/integration/pyramid.py

```diff
@@ -127,11 +127,11 @@ class BaseplateConfigurator(object):
 
     """
 
-    def __init__(self, baseplate, trust_trace_headers=False,
+    def __init__(self, baseplate, trust_trace_headers=None,
                  edge_context_factory=None, header_trust_handler=None):
         self.baseplate = baseplate
-        self.trust_trace_headers = trust_trace_headers
-        if self.trust_trace_headers:
+        self.trust_trace_headers = bool(trust_trace_headers)
+        if self.trust_trace_headers is not None:
```

Comment:
this needs to be `if trust_trace_headers is not None`, `self.trust_trace_headers` is always a boolean.

Reply:
good catch